### PR TITLE
Adding workflow tests and fix installation in documentation

### PR DESCRIPTION
**Pull request by wellwelwel**
Hi @aashutoshrathi, I would like to contribute by automating existing tests for every new submitted **PR** and for pushes to `master` branch.

> Having millions of downloads, it's good to always make sure nothing is broken with new changes (except when intentionally).

---

About the `package-lock.json` (*2,246 additions by performing `npm i`*), it's necessary to install the correct packages and perform the **CI** tests.
> npm ci

---

Also, I noticed two errors in `README.md`:

1)
https://github.com/aashutoshrathi/word-wrap/blob/84981421a9ff67bf7604a1183c719b9bfe053994/README.md#L10

2)
https://github.com/aashutoshrathi/word-wrap/blob/84981421a9ff67bf7604a1183c719b9bfe053994/README.md#L16

- Then, I just fixed it to your **npm** package.

---

Feel free to ask anything 🚀

### CODE REVIEW

**Comment by wellwelwel** [comment]
Just complementing, for now, it wasn't possible to test versions earlier than **NodeJS 14**, because **Mocha** requires the **NodeJS 14.0.0** or higher.

**Comment by wellwelwel** [comment]
Another point I haven't touched on is `trimEnd`:

https://github.com/aashutoshrathi/word-wrap/blob/84981421a9ff67bf7604a1183c719b9bfe053994/index.js#L10

---

@aashutoshrathi, I don't think a *polyfill* is necessary, just set the engine in [`package.json`](https://github.com/aashutoshrathi/word-wrap/blob/84981421a9ff67bf7604a1183c719b9bfe053994/package.json#L32) to `>=10.3.0`. But I leave it to you decide.

**Comment by aashutoshrathi** [comment]
> Another point I haven't touched on, is `trimEnd`:
> 
> https://github.com/aashutoshrathi/word-wrap/blob/84981421a9ff67bf7604a1183c719b9bfe053994/index.js#L10
> 
> @aashutoshrathi, I don't think a polyfill is necessary, just set the engine in [`package.json`](https://github.com/aashutoshrathi/word-wrap/blob/84981421a9ff67bf7604a1183c719b9bfe053994/package.json#L32) to `>=10.3.0`. But I leave it to you decide.

But I want to be sure that if we do the polyfill, will we be vulnerability free? As I think the polyfill uses regex

**Comment by aashutoshrathi** [comment]
So, it is okay to publish without `package-lock.json`, but we need that for CI it, right?

**Review by aashutoshrathi — APPROVED** [review]

**Comment by aashutoshrathi** [comment]
Hey! @wellwelwel Can you point this PR to `v1.2.7` and take a pull of the same branch once.

File: README.md
Comment (aashutoshrathi) [inline README.md:1]: Also, can you add the [CI Badge](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge) too?

**Comment by wellwelwel** [comment]
> So, it is okay to publish without `package-lock.json`, but we need that for CI it, right?

I'm away from PC now, but the `package-lock.json` probably will be ignored because the `files` field into `package.json`.

**Comment by wellwelwel** [comment]
> Hey! @wellwelwel Can you point this PR to v1.2.7 and take a pull of the same branch once.

Alright, but after I wake up 🙋🏻‍♂️

**Comment by wellwelwel** [comment]
Closing in favor of:
- #8

**Pull request closed without merging**